scripts/mysql/tls_mgm-create.sql

```diff
@@ -6,14 +6,14 @@ CREATE TABLE tls_mgm (
     method CHAR(16),
     verify_cert INT(1),
     require_cert INT(1),
-    certificate CHAR(255),
-    private_key CHAR(255),
+    certificate BLOB,
```

Comment:
Did you manually change the values here? You should update the database [schema](https://github.com/OpenSIPS/opensips/blob/master/db/schema/tls_mgm.xml) and re-generate it for all backends (`make dbschema`).